tracing/zipkin/zipkin.go

```diff
@@ -259,6 +312,12 @@ func FromContext(ctx context.Context) (*Span, bool) {
 	return span, true
 }
 
+// CollectorToContext stores a Zipkin collector to be used when annotating
+// resources using their own child spans with NewChildSpan.
+func CollectorToContext(ctx context.Context, c Collector) context.Context {
+	return context.WithValue(ctx, CollectorContextKey, c)
+}
```

Comment:
Hmm, this doesn't appear to be used, and I don't totally understand the comment. Can you explain more?